src/services/authService.js

```diff
@@ -255,4 +255,37 @@ export default class AuthService {
 
     return 'User Role Assigned Successfully';
   }
+
+  /**
+   * @param {object} req reqest object
+   * @return {string} - success message;
+   */
+  static async rememberUserProfile(req) {
+    const { rememberProfile } = req.body;
+    const { id } = req.user;
+    const getUser = await User.findOne({ where: { id } });
+
+    if (!getUser) error('User not found');
+
+    await User.update({ rememberProfile }, { where: { id } });
+
+    return 'Remember Profile Information Successfully Updated';
+  }
+
+  /**
+   * @param {object} req reqest object
+   * @return {string} - success message;
+   */
+  static async getRememberProfile(req) {
+    const { id } = req.user;
```

Comment:
You can destructure req.user as an argument like this `static async getRememberProfile({ user: { id } })` that way your code is shorter and cleaner and it follows the team's convention. Please also effect this in other necessary places